ledger/src/shred.rs

```diff
@@ -910,12 +855,8 @@ pub fn max_entries_per_n_shred_last_or_not(
     is_last_in_slot: bool,
 ) -> u64 {
     // Default 32:32 erasure batches yields 64 shreds; log2(64) = 6.
-    let merkle_variant_unsigned = Some((
-        /*proof_size:*/ 6, /*chained:*/ true, /*resigned:*/ false,
-    ));
-    let merkle_variant_signed = Some((
-        /*proof_size:*/ 6, /*chained:*/ true, /*resigned:*/ true,
-    ));
+    let merkle_variant_unsigned = Some((/*proof_size:*/ 6, /*resigned:*/ false));
```

Comment:
Do we need Some/None semantics here at all now that all shreds are merkle? Maybe another PR material.